packages/larva-js/src/interface/ProfileFilter/__test__/ProfileFilter.test.js

```diff
@@ -1,7 +1,7 @@
 const assert = require( 'assert' );
 
-import initProfileFilter from './index';
-import ProfileFilter from './ProfileFilter';
+import initProfileFilter from '../index';
```

Comment:
Updated the location of this test per discussion on component test location standards, and that tests should be next to the component.